exporters/otlp/otlplog/otlploghttp/internal/transform/log.go

```diff
@@ -18,18 +19,26 @@ import (
 	"go.opentelemetry.io/otel/sdk/log"
 )
 
-// ResourceLogs returns an slice of OTLP ResourceLogs generated from records.
-func ResourceLogs(records []log.Record) []*lpb.ResourceLogs {
+var resourceLogsPool = sync.Pool{
```

Comment:
These pool values escape this package. It is likely a better design to have pooled values passed to `ResourceLogs` instead so leaks are not incurred.